qontract_api/tests/integrations/quay_robot_accounts/test_quay_robot_accounts_router.py

```
@@ -0,0 +1,135 @@
+"""Unit tests for quay-robot-accounts router endpoints."""
```

Ruff (0.16.2)

[warning] File `qontract_api/tests/integrations/quay_robot_accounts/test_quay_robot_accounts_router.py` is part of an implicit namespace package. Add an `__init__.py`.

(INP001)